src/Microsoft.OpenApi/Extensions/EnumExtensions.cs

```diff
@@ -42,5 +43,88 @@ public static string GetDisplayName(this Enum enumValue)
             var attribute = enumValue.GetAttributeOfType<DisplayAttribute>();
             return attribute == null ? enumValue.ToString() : attribute.Name;
         }
+
+        /// <summary>
+        /// Gets the enum display for name <see cref="ParameterStyle" /> without the use of reflection.
+        /// </summary>
+        /// <param name="parameterStyle">The enum value.</param>
+        /// <returns>The display string to use.</returns>
+        internal static string GetDisplayName(this ParameterStyle parameterStyle) => parameterStyle switch
```

Comment:
should we mark the non type specific methods as obsolete?

Reply:
Done! This ended up catching some invocation sites that I missed.